src/signals/settings/users/containers/Overview/index.js

```diff
@@ -103,6 +129,14 @@ export const UsersOverviewContainer = ({ pageSize, userCan }) => {
           <Column span={12}>
             <DataView
               headers={columnHeaders}
+              filters={[
+                (<StyledSearchbar
+                  label="Zoek op emailadres"
```

Comment:
I don't see the label anywhere when I load the user overview page. Does it do anything?